### PR TITLE
Force and dissipation

### PR DESCRIPTION
**Pull request by esfigueroa**
New pull request

### CODE REVIEW

File: output/data_plotting.py [hunk 1]
```diff
@@ -1,48 +1,81 @@
...
+matplotlib.rcParams.update({'font.size': 15})  # Size of font of plot
 
-# Load data
-phi_t_list = []
-inner_phi = np.zeros((var.D, var.D))
-first_row_phi = []
-for p in range(1, var.number_iterations + 1):
+# Change the current directory:
+# os.chdir('C:\\Users\\Elram Figueroa\\PycharmProjects\\sine-gordon\\output\\test_dx_0.5_dt_0.01_D_201_F_0.1')
+os.chdir('C:\\Users\\Elram Figueroa\\PycharmProjects\\sine-gordon\\output\\test_dx_0.5_dt_0.01_D_201_F_-0.1')
```
Comment (garcia-science) [inline output/data_plotting.py:13]: Put this path into variables file

File: output/data_plotting.py [hunk 2]
```diff
@@ -1,48 +1,81 @@
...
+
+# Load data soliton
+phi_t_list = []  # This list will contain all data from t_i.txt
+inner_phi = np.zeros((var.D, var.D))  # Initializing the matrix
+# first_row_phi = [] # If we need one row only of every matrix, use this
+for p in range(0, len(center_neg - 1)):
     inner_phi = np.loadtxt('t_' + str(p) + '.txt')
     phi_t_list.append(inner_phi)
-    first_row_phi.append((inner_phi[1]))
+    # first_row_phi.append((inner_phi[1]))
```
Comment (garcia-science) [inline output/data_plotting.py:27]: Put all this code inside a method, try to think how you can build an abstract method to plot in order to use each time as you need.

File: output/data_plotting.py [hunk 3]
```diff
@@ -1,48 +1,81 @@
...
 fig1 = plt.figure(1)
+plt.plot(x_axes_center, center, linewidth=2, label='$f=0.1$, $\gamma=0.1$')
+plt.plot(x_axes_center, center_neg, linewidth=2, label='$f=-0.1$, $\gamma=0.1$')
+plt.xlabel(r'$Time$', fontsize=17)
+plt.ylabel(r'$\dot{\psi}(\delta)$', fontsize=17)
+plt.xticks(fontsize=16)
+plt.yticks(fontsize=16)
+plt.legend(shadow=True, fontsize='x-large')
+plt.show()
+"""
```
Comment (garcia-science) [inline output/data_plotting.py:52]: Same, build a method on order to plot, like receive some data and build the plot as you want.

File: output/plot_soliton_center.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+# phase field equations
+# eq2 = r'\begin{eqnarray*}' + \
+# r'\mathcal{F} &=& \int f\left( \phi, c \right) dV, \\ ' + \
+# r'\frac{ \partial \phi } { \partial t } &=& -M_{ \phi } ' + \
+# r'\frac{ \delta \mathcal{F} } { \delta \phi }' + \
+# r'\end{eqnarray*}'
+# plt.text(0.18, 0.18, eq2, {'color': 'C0', 'fontsize': 16})
+
+plt.show()
```
Comment (garcia-science) [inline output/plot_soliton_center.py]: Build a method for this code.

**Review by garcia-science — COMMENTED** [review]
Change the code in order to build methods. You need to start to thinking in modular way. After you can use or reuse the same method in order to plot. 
Let´s discuss later, and I will help you.
Oliver

**Review by esfigueroa — COMMENTED** [review]
I created a new modular section: plotting. I followed the indications of Oliver, and I feel happy (I maked a scrip that it make the video of once).

**Review by esfigueroa — COMMENTED** [review]
I added a script to verify the force (plot)

**Review by esfigueroa — COMMENTED** [review]
Plots codes and anything else

**Review by esfigueroa — COMMENTED** [review]
Plotting and module of calculate center of soliton

**Pull request merged**